spec/spec_helper.rb

```diff
@@ -5,6 +5,8 @@
 require 'io/console'
 require 'pry-byebug'
 require 'random_word'
+require 'rubyXL'
+require 'rubyXL/convenience_methods/cell'
```

Comment:
why do we need this?

Reply:
The `rubyXL` gem is  the Excel spreadsheet

Reply:
I see why we need rubyXL, but why this second require?

Reply:
I see that this is for `change_contents` in XlsxHelpers.  Can we move that into that module and add a comment about that?

Reply:
Done. Moved the `rubyXL/convenience_methods/cell` require to the xlsx_helpers with a comment.